lib/teams_collection.rb

```diff
@@ -12,4 +15,8 @@ def generate_objects_from_csv(csv_teams_path)
     end
     objects
   end
+
+  def count_of_teams
+    @teams.count {|team| team.team_id.length}
```

Comment:
Here's a good place to utilize the "total_unique" method that I wrote, if you want to come back to this after putting it in a module.